tikv/raftstore/fsm_apply.go

```diff
@@ -1,12 +1,46 @@
 package raftstore
 
 import (
+	"container/list"
+	"github.com/ngaut/unistore/tikv/import"
+	"github.com/ngaut/unistore/util"
 	"github.com/pingcap/kvproto/pkg/eraftpb"
 	"github.com/pingcap/kvproto/pkg/import_sstpb"
 	"github.com/pingcap/kvproto/pkg/metapb"
+	"github.com/pingcap/kvproto/pkg/raft_cmdpb"
 	rspb "github.com/pingcap/kvproto/pkg/raft_serverpb"
+	"github.com/uber-go/atomic"
 )
 
+type PendingCmd struct {
+	index uint64
+	term  uint64
+	cb    Callback
+}
+
+type PendingCmdQueue struct {
+	normals    *list.List
```

Comment:
We can simply use the slice for rust queue.

Reply:
ok.